rust_snuba/src/processors/spans.rs

```diff
@@ -216,21 +222,6 @@ impl TryFrom<FromSpanMessage> for Span {
             .map(|(k, v)| (k, v.value))
             .unzip();
 
-        if let Some(http_method) = from.sentry_tags.http_method {
-            tag_keys.push("http.method".into());
-            tag_values.push(http_method);
-        }
-
-        if let Some(status_code) = from.sentry_tags.status_code {
-            tag_keys.push("status_code".into());
-            tag_values.push(status_code);
-        }
-
-        if let Some(transaction_method) = from.sentry_tags.transaction_method {
-            tag_keys.push("transaction.method".into());
-            tag_values.push(transaction_method);
-        }
```

Comment:
These are not meant to be in tags, just sentry_tags, the Python processor had incorrect behavior.